frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs

```diff
@@ -63,6 +64,10 @@ public void OnStartLine(HttpMethod method, HttpVersion version, Span<byte> targe
                 {
                     requestType = RequestType.Fortunes;
                 }
+                else if (Paths.SingleQuery.Length <= path.Length && path.StartsWith(Paths.SingleQuery))
```

Comment:
Should we order these length tests from the shortest to the biggest paths?

Reply:
Done

Reply:
Do you guys see performance difference compared to simple string comparison?

Reply:
The Length test is a simple memory test, the StartsWith is a non-inlining call - it probably won't be a huge factor, but why make the call for everything its not?